src/gradata/enhancements/behavioral_extractor.py

```
@@ -0,0 +1,497 @@
+    REPLACEMENT_FACTUAL = auto()  # Numbers/dates/URLs corrected
+    REORDER = auto()              # Same content, different order
+    FORMAT_CHANGE = auto()        # Structure changed (prose→list, etc.)
+    TRUNCATION = auto()           # Significantly shortened (>35% reduction)
+    EXPANSION = auto()            # Significantly expanded (>50% growth)
+    CONSTRAINT_ADDITION = auto()  # New always/never/must rule
+
+
+@dataclass
+class ArchetypeMatch:
+    archetype: Archetype
+    confidence: float  # 0.0–1.0
```

Ruff (0.15.9)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)